cloudevents/sdk/http_events.py

```diff
@@ -0,0 +1,217 @@
+"""This module maintains HTTP/HTTPS binary cloudevents
+
+This module currently only supports binary events and http/https
+formatted requests. You're able to use the internal Event class
+to emit cloudevent formatted HTTP/HTTPS requests to API endpoints,
+and to retrieve binary cloudevent formatted HTTP/HTTPS headers and data.
+
+Example:
+    To create an event which interacts with cloudevents,
+    do the following:
+        > from cloudevents.sdk.http_events import Event
+        > Event(headers=headers, data=data)
+        > print(Event)
+
+Attributes:
+    Event(dict, any, bool?, typing.Callable?): module level class which
+        abstracts internal marshalling and event data structures. Provides
+        an easy to use interface into the cloudevents sdk
+
+TODO:
+    SUPPORT structured calls
+    Test application/json content-type
+    Support other HTTP Methods in emit_binary_event & emit_structured_event
+    Refactor content-type to datacontenttype
+    Implement testing
+
+Licensing:
+    All Rights Reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may
+    not use this file except in compliance with the License. You may obtain
+    a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+    License for the specific language governing permissions and limitations
+    under the License.
```

Comment:
Use a license comment like this:

https://github.com/cloudevents/sdk-python/blob/master/cloudevents/sdk/event/base.py#L1